climakitae/data_loaders.py

```diff
@@ -60,13 +29,13 @@ def _open_and_concat(cat, file_list, selections, geom):
     for one_file in file_list:
         with cat[one_file].to_dask() as data:
             source_id = data.attrs["source_id"]
-            if selections.variable == "precipitation (total)":
+            if selections.variable not in ("precipitation (total)", "wind 10m magnitude"):
+                data = data[selections.variable]
+            elif selections.variable == "precipitation (total)":
                 pass
             elif selections.variable == "wind 10m magnitude":
```

Comment:
Sorry, I probably wasn't clear, but these elifs aren't needed with the change to `if selections.variable not in ("precipitation (total)", "wind 10m magnitude"):`

Reply:
I know they're not needed, but they will be needed later when there's something there other than 'pass', which is one of the cards on my to-do.